pkgs/development/python-modules/pipx/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, userpath
+, argcomplete
+, packaging
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "pipx";
+  version = "0.15.5.0";
+
+  disabled = pythonOlder "3.6";
+
+  # no tests in the pypi tarball, so we directly fetch from github
+  src = fetchFromGitHub {
+    owner = "pipxproject";
+    repo = pname;
+    rev = version;
+    sha256 = "13z032i8r9f6d09hssvyjpxjacb4wgms5bh2i37da2ili9bh72m6";
+  };
+
+  propagatedBuildInputs = [ userpath argcomplete packaging ];
+
+  postPatch = ''
```

please add comment about why this is necessary and when i can be removed
```suggestion
  # avoid inconclusive venv assertion, see https://github.com/pipxproject/pipx/pull/477
  # remove after PR is merged
  postPatch = ''
```